aws/resource_aws_sns_topic.go

```diff
@@ -176,35 +156,223 @@ func resourceAwsSnsTopicCreate(d *schema.ResourceData, meta interface{}) error {
 
 	d.SetId(*output.TopicArn)
 
-	for terraformAttrName, snsAttrName := range SNSAttributeMap {
-		if d.HasChange(terraformAttrName) {
-			_, terraformAttrValue := d.GetChange(terraformAttrName)
-			err := updateAwsSnsTopicAttribute(d.Id(), snsAttrName, terraformAttrValue, snsconn)
-			if err != nil {
-				return err
-			}
+	// update mutable attributes
+	if d.HasChange("application_failure_feedback_role_arn") {
+		_, v := d.GetChange("application_failure_feedback_role_arn")
```

Comment:
Nit: These will be fixed later on with #12085, but only using the second value of `d.GetChange()` can be replaced with `d.Get()`. This is no different than before so it is fine to leave for now. 👍